<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dexcsCfdMesh</class>
 <widget class="QWidget" name="dexcsCfdMesh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>1330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dexcsCFD Mesh</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mesh case</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_8">
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_meshTool"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_dimension"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="l_dimension">
       <property name="text">
        <string>dimension</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="l_meshTool">
       <property name="text">
        <string>Mesh Tool</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0" columnstretch="0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="pb_edit_mesh">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pb_write_mesh">
       <property name="text">
        <string>Write mesh case</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="l_scaleToMeter">
       <property name="text">
        <string>Scale to Meter:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="if_scaleToMeter">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mesher</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0">
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <property name="spacing">
      <number>8</number>
     </property>
     <item row="1" column="1">
      <widget class="QPushButton" name="pb_stop_mesh">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pb_run_mesh">
       <property name="text">
        <string>Run mesher</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mesh verification</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_7">
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <property name="horizontalSpacing">
      <number>14</number>
     </property>
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <item row="9" column="0">
      <widget class="QPushButton" name="pb_paraview">
       <property name="text">
        <string>Paraview</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="pb_checkmesh">
       <property name="text">
        <string>CheckMesh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mesh Parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="cfMeshSpecificProperties">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <property name="spacing">
       <number>8</number>
      </property>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gl_actions">
        <item row="1" column="0">
         <widget class="QLabel" name="l_workflowControls">
          <property name="text">
           <string>workflowControls(stopAfter):</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="l_time">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <property name="horizontalSpacing">
           <number>14</number>
          </property>
          <property name="verticalSpacing">
           <number>8</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="l_featureAngle">
            <property name="text">
             <string>Feature Angle(deg):</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QFrame" name="optimizer_frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item row="2" column="0">
              <layout class="QGridLayout" name="gridLayout_7">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>3</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <item row="2" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label_maxNumIterations">
                   <property name="text">
                    <string>maxNumIterations:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="3" column="0" alignment="Qt::AlignRight">
                <widget class="QSpinBox" name="if_maxNumIterations">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_maxAllowedThickness">
                   <property name="text">
                    <string>maximum allowed thickness:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="4" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>7</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_featureSizeFactor">
                   <property name="text">
                    <string>featureSizeFactor:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="5" column="0" alignment="Qt::AlignRight">
                <widget class="QDoubleSpinBox" name="if_featureSizeFactor">
                 <property name="decimals">
                  <number>6</number>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>0.400000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.050000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label_reCalculateNormals">
                   <property name="text">
                    <string>reCalculateNormals:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="check_reCalculateNormals">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label_nSmoothNormals">
                   <property name="text">
                    <string>Number of iterations:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0" alignment="Qt::AlignRight">
                <widget class="QSpinBox" name="if_nSmoothNormals">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item row="8" column="0" alignment="Qt::AlignRight">
                <widget class="QDoubleSpinBox" name="if_relThicknessTol">
                 <property name="decimals">
                  <number>6</number>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.001000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="l_max">
            <property name="text">
             <string>Base element size:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="check_optimiseLayer">
            <property name="text">
             <string>optimiseLayer</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="if_featureAngle">
            <property name="decimals">
             <number>6</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="check_keepCells">
            <property name="text">
             <string>keepCellsIntersectingBoundary</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="Gui::InputField" name="if_max">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string extracomment="Select 0 to use default value"/>
            </property>
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="singleStep" stdset="0">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum" stdset="0">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="unit" stdset="0">
             <string notr="true">mm</string>
            </property>
            <property name="format" stdset="0">
             <string>g</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" alignment="Qt::AlignRight">
         <widget class="QComboBox" name="cb_workflowControls">
          <property name="toolTip">
           <string>workflowControls:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="snappySpecificProperties">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="2" column="1">
       <widget class="Gui::InputField" name="if_pointInMeshY"/>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="if_edgerefine"/>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="if_cellsbetweenlevels"/>
      </item>
      <item row="2" column="0">
       <widget class="Gui::InputField" name="if_pointInMeshX"/>
      </item>
      <item row="5" column="2">
       <widget class="QRadioButton" name="radio_explicit_edge_detection">
        <property name="text">
         <string>Explicit</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">shmButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="Gui::InputField" name="if_pointInMeshZ"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Edge detection</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pb_searchPointInMesh">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="radio_implicit_edge_detection">
        <property name="text">
         <string>implicit</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">shmButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>No of cells between levels</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Relative edge refinement</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Base element Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Point in mesh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="Gui::InputField" name="if_shm_max"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="te_output">
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::InputField</class>
   <extends>QLineEdit</extends>
   <header>Gui/InputField.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pb_run_mesh</tabstop>
  <tabstop>pb_stop_mesh</tabstop>
  <tabstop>if_max</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="shmButtonGroup"/>
 </buttongroups>
</ui>
